src/commands/cloud-run/README.md

```diff
@@ -0,0 +1,38 @@
+## Troubleshooting Cloud Run Instrumentation
+
+To troubleshoot issues you may be encountering with Datadog monitoring on your Cloud Run services, use the `datadog-ci cloud-run flare` command in the root of your project directory. This command collects important data about a Cloud Run service, such as environment variables and configuration information. These files will be submitted to Datadog support via a ticket matching the provided Zendesk case ID.
+
+**Examples**
+```bash
+# Collect and send files to Datadog support for a single service
+datadog-ci cloud-run -s <service> -p <project> -r <region/location> -c <case-id> -e <email-on-case-id>
+
+# Include recent logs
+datadog-ci cloud-run -s <service> -p <project> -r <region/location> -c <case-id> -e <email-on-case-id> --with-logs
+
+# Dry run: collect data, but don't send to Datadog support
+datadog-ci cloud-run -s <service> -p <project> -r <region/location> -c <case-id> -e <email-on-case-id> --with-logs --dry-run
+```
+
+**Arguments**
+
+| Argument              | Shorthand | Description                                                                                                                 | Default |
+|-----------------------|-----------|-----------------------------------------------------------------------------------------------------------------------------|---------|
+| `--service`           | `-s`      | The name of the Cloud Run service.                                                                                          |         |
+| `--project`           | `-p`      | The name of the Google Cloud project within which the Cloud Run service is hosted.                                          |         |
+| `--region`            | `-r`      | The region where the Cloud Run service is hosted.                                                                           |         |
+| `--case-id`           | `-c`      | The Datadog case ID to send the files to.                                                                                   |         |
+| `--email`             | `-e`      | The email associated with the specified case ID.                                                                            |         |
+| `--with-logs`         |           | Collect recent logs for the specified service.                                                                              | `false` |
+| `--start` and `--end` |           | Define a time range in milliseconds since the Unix Epoch to gather logs within that range. (`--with-logs` must be included) |         |
```

Comment:
```suggestion
| `--start` and `--end` |           | Define a time range in milliseconds since the Unix Epoch to gather logs within that range. (`--with-logs` must be included.) |         |
```

Reply:
I'm getting slightly tripped up by the wording "since the Unix Epoch to gather". Can you clarify what that means?

Reply:
"Milliseconds since Unix Epoch" represents a timestamp. The user can specify two times:`--start` and `--end` to only get logs within that time range. I have updated the wording a bit to be more clear, WDYT?